platform/viewer/cypress/integration/common/OHIFDownloadSnapshotFile.spec.js

```diff
@@ -0,0 +1,188 @@
+    cy.expectMinimumThumbnails(5);
+  });
+
+  beforeEach(() => {
+    cy.openDownloadImageModal();
+  });
+
+  afterEach(() => {
+    cy.get('[data-cy="close-button"]')
+      .scrollIntoView()
+      .click();
+  });
+
+  it('checks displayed information for Tablet experience', function() {
+    // Set Tablet resolution
+    cy.viewport(1000, 660);
+    // Visual comparison
+    cy.screenshot('Download Image Modal - Tablet experience');
+    cy.percyCanvasSnapshot('Download Image Modal - Tablet experience');
+    //Check if all elements are displayed
+    cy.get('.OHIFModal')
+      .as('downloadImageModal')
+      .should('contain.text', 'Download High Quality Image');
+    // Check input fields
+    cy.get('.file-info-container')
+      .should('contain.text', 'Image width (px)')
+      .should('contain.text', 'Image height (px)')
+      .and('contain.text', 'File name')
+      .and('contain.text', 'File type')
+      .and('contain.text', 'Show Annotations');
+    cy.get('.file-type')
+      .find('select')
+      .select('png')
+      .should('have.value', 'png')
+      .select('jpg')
+      .should('have.value', 'jpg');
+    // Check image preview
+    cy.get('.preview').should('contain.text', 'Image Preview');
+    cy.get('[data-cy="viewport-preview-img"]')
+      .should('have.attr', 'src')
+      .and('include', 'data:image');
+    // Check buttons
+    cy.get('[data-cy="cancel-btn"]')
+      .scrollIntoView()
+      .should('have.text', 'Cancel');
+    cy.get('[data-cy="download-btn"]')
+      .scrollIntoView()
+      .should('have.text', 'Download');
+  });
+
+  it('checks displayed information for Desktop experience', function() {
+    // Set Desktop resolution
+    cy.viewport(1750, 720);
+    // Visual comparison
+    cy.screenshot('Download Image Modal - Desktop experience');
+    cy.percyCanvasSnapshot('Download Image Modal - Desktop experience');
+    //Check if all elements are displayed
+    cy.get('.OHIFModal')
+      .as('downloadImageModal')
+      .should('contain.text', 'Download High Quality Image');
+    // Check input fields
+    cy.get('.file-info-container')
+      .should('contain.text', 'Image width (px)')
+      .should('contain.text', 'Image height (px)')
+      .and('contain.text', 'File name')
+      .and('contain.text', 'File type')
+      .and('contain.text', 'Show Annotations');
+    cy.get('.file-type')
+      .find('select')
+      .select('png')
+      .should('have.value', 'png')
+      .select('jpg')
+      .should('have.value', 'jpg');
+    // Check image preview
+    cy.get('.preview').should('contain.text', 'Image Preview');
+    cy.get('[data-cy="viewport-preview-img"]')
+      .should('have.attr', 'src')
+      .and('include', 'data:image');
+    // Check buttons
+    cy.get('[data-cy="cancel-btn"]')
+      .scrollIntoView()
+      .should('have.text', 'Cancel');
+    cy.get('[data-cy="download-btn"]')
+      .scrollIntoView()
+      .should('have.text', 'Download');
+  });
```

Comment:
Can we keep all of the `text` checks for `Desktop`, but only do a visual/snapshot test for the other resolutions?
If we eventually opt to have different templates at certain breakpoints, we can get more granular.

One thing to verify is that updating one field's value results in the other's updating as well.